pf4j-web-demo/demo/pom.xml

```diff
@@ -14,7 +14,7 @@
 	</modules>
 
 	<properties>
-		<pf4j.version>2.0.0</pf4j.version>
+		<pf4j.version>2.2.0</pf4j.version>
```

Comment:
here it's a tab indentation (8 chars). I prefer 4 chars as indentation. Please also review the readme file for same problem.

Reply:
As you see in diff, it was carried over from previous, but i can change to 4 chars

Reply:
Reformated the document with 4 char settings, now it shows like full file is modified.